src/cli/commands/config.js

```diff
@@ -43,26 +41,17 @@ module.exports = {
 
       if (!value) {
         // Get the value of a given key
-
-        if (utils.isDaemonOn()) {
-          return ipfs.config.get(key, (err, config) => {
-            if (err) {
-              log.error(err)
-              throw new Error('failed to read the config')
-            }
-
-            console.log(config.Value)
-          })
-        }
-
-        ipfs.config.get((err, config) => {
+        ipfs.config.get(key, (err, value) => {
           if (err) {
             log.error(err)
             throw new Error('failed to read the config')
           }
 
-          const value = get(config, key)
-          console.log(value)
+          if (typeof value === 'object') {
+            console.log(JSON.stringify(value, null, 2))
+          } else {
+            console.log(value)
+          }
```

Comment:
Same here as below, we should probably try using the logger rather than console.log